tests/bud.bats

```diff
@@ -1620,15 +1621,24 @@ function _test_http() {
   gitrepo=github.com/projectatomic/nulecule-library
   run_buildah build $WITH_POLICY_JSON -t ${target} "${gitrepo}"
   run_buildah from ${target}
+  # Test where context is repourl.git#commit
+  gitrepo=github.com/projectatomic/nulecule-library#fa1c7025262f5d33ba6339dd70c1163c9c26108f
+  run_buildah build $WITH_POLICY_JSON -t ${target} "${gitrepo}"
+  run_buildah from ${target}
```

Comment:
Try to use a different name for the image this line builds than was used for the previous image built in this test.

Reply:
this is a new test now using local git repo